scalameter-core/src/main/scala/org/scalameter/InvocationCountMatcher.scala

```diff
@@ -0,0 +1,123 @@
+package org.scalameter
```

Comment:
Remember the convention:
- filenames are named after the top-level class and companion object that reside in them
- if there are more top-level classes, filename is lowercase, words divided with dashes

In this case, you should go for the first variant, as most other classes are simple. You can move most of the other classes into `InvocationCountMatcher`.